compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_libcdep.cpp

```diff
@@ -274,6 +280,8 @@ class LLVMSymbolizerProcess final : public SymbolizerProcess {
     const char* const kSymbolizerArch = "--default-arch=arm64";
 #elif defined(__arm__)
     const char* const kSymbolizerArch = "--default-arch=arm";
+#elif defined(__powerpc__) && __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
+    const char* const kSymbolizerArch = "--default-arch=powerpc";
```

Comment:
This is incorrect. `__powerpc__` is also set when `__powerpc64__` is set. This change prevents reaching the `--default-arch=powerpc64` case.